File: C++/Inheritance/Emp.cpp
#include"Emp.h"
Emp::Emp(){empid=0;salary=0;}
Emp::Emp(int eid,const char*nm,int d,int m,int y,double sal):Person(nm,d,m,y)
{
    this->empid=eid;
    this->salary=sal;
}
void Emp::display()
{
    cout<<empid<<endl;
    Person::display();
    cout<<endl;
    cout<<salary;
}

File: Collections/src/treeSet/Emp.java
package treeSet;

public class Emp {
	public int empid;
	public String name;
	public float salary;
	public int dept;
	public Emp()
	{
		name="NA";
		salary=0f;
	}
	
	public Emp(String name, float salary) {
	
		this.name = name;
		this.salary = salary;
	}

	public Emp(int empid,String name, float salary,int dept) {
		this.empid=empid;
		this.name = name;
		this.salary = salary;
		this.dept=dept;
	}
	public String toString()
	{
		return "empid->"+empid+"name->"+name+" salary->"+salary+" dept->"+dept+"\n";
	}
	
}
